v2/internal/runner/runner.go

```diff
@@ -138,10 +138,12 @@ func New(options *types.Options) (*Runner, error) {
 	dupeCount := 0
 
 	// Handle single target
```

Comment:
please remove the "Handle single target" comment, because it's not the case anymore

Reply:
the comment is still the same